mappings/charging_station.yml

```diff
@@ -1,18 +1,17 @@
 name: Stations de recharge de véhicules électriques
 overpass: nwr["amenity"="charging_station"](area.searchArea);
 mapping:
-  covered: covered|bool
   opening_hours: opening_hours
   parking_fee: "parking:fee"
-  start_date: start_date
   owner: owner
   operator: operator
   website: website
-  chademo: "socket:chademo|bool"
+  capacity: capacity
```

Comment:
capacity|int ? (To ensure an int value in the JSON output)

Reply:
fixed